sdk/client.go

```diff
@@ -17,6 +17,7 @@ type Client interface {
 	Checker(ctx context.Context, check *health.CheckState) error
 	Health() *healthcheck.Client
 	PostJob(ctx context.Context, headers Headers) (models.Job, error)
+	PostTasksCount(ctx context.Context, headers Headers, jobID string) (models.Task, error)
```

Comment:
I think it would be better just to call it 'PostTask' otherwise it could get confused with the PUT endpoint that will be used to update the count of tasks. Also, it needs another parameter for the request body e.g. 
```suggestion
	PostTask(ctx context.Context, headers Headers, jobID string, taskToCreate models.TaskToCreate) (models.Task, error)
```